style.css

```diff
@@ -84,31 +96,204 @@ body{
 
     margin-top:10px;
 }
-.collab-btn:hover,.collab-btn:focus{
+.collab-btn:hover,.collab-btn:focus,.cvbtn:hover,.cvbtn:focus{
     background-color: #fff;
     color:var(--primary-color);
     border: 1px solid var(--primary-color);
 }
-.collab-btn:active{
+.collab-btn:active,.cvbtn:active{
     border: 2px solid var(--primary-color);
 }
-@media(max-width:375px){
-    .navbar{
-        background-image:url("./img/image_geometry_1.svg");
-        background-repeat:no-repeat;
-    }
-    .author{
-        width:80%;
-        height:100px;
-    }
-    .collab-btn{
-        width:100%;
-        height: 7vh;
-    }
-    .headline{
-        background-image:url("./img/image_geometry_2.svg");
-        background-repeat:no-repeat;
-        background-size:100%;
-        background-position:50% 90%;
-    }
+/* END HEADLINE SECTION */
+/* START WORK SECTION */
+.works{
+    margin:30px 0;
+}
+.title h2{ 
+    color:var(--primary-color);
+    font-family: 'Playfair Display', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 50px;
+    line-height: 100%;
+    margin:0 0 35px 20px;
+}
+.project-cards{
+    background-color:var(--primary-color);
+    
+    flex-direction: column;
+}
+.pjimgs img{
+    width:100%;
+    height:100%;
+    flex:1 1 55%;
+}
+.pjcontent{
+    background-color:var(--primary-color);
+    color:#fff;
+
+    padding:20px;
+    flex:1 1 45%;
+}
+.pjname{
+    font-family: 'Poppins';
+    font-style: normal;
+    font-weight: 300;
+    font-size: 20px;
+    line-height: 80%;
+
+    margin:0;
+    padding:0;
+}
+.pjcontent ul{
+    width:90%;
+
+    margin:20px 0;
+    padding:0;  
+}
+.pjcontent ul li{
+    background-color:#98DEDE;
+    color:var(--primary-color);
+    list-style-type: none;
+
+    text-align:center;
+    font-family: 'Poppins';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 10.3334px;
+    line-height: 80%;
+    padding:7px 11px;
+}
+.pjcontent ul li:nth-of-type(1){
+    flex:0.5 2 8%;
+}
+.pjcontent ul li:nth-of-type(2){
+    flex:1 0 12%;
+    padding:7px 0;
+    margin:0 10px;
+}
+.pjcontent ul li:nth-of-type(3){
+    flex:0.5 2 10%;
+}
+.pjbtn{
+    width:60%;
+    background: var(--primary-color);
+    font-size: 16px;
+    font-family: 'Poppins';
+    font-weight: 300;
+    border: 1px solid #FFFFFF;
+    backdrop-filter: blur(0.5px);
+    padding:10px;
+}
```

Comment:
As requested by the previous reviewer, please make sure to add the corresponding effect to these buttons:
![image](https://user-images.githubusercontent.com/42799579/205349707-6a53294d-83d1-4f9a-b39c-efe8bae53862.png)

You should use these effects:
![image](https://user-images.githubusercontent.com/42799579/205349809-49b4ab4d-0b6e-4348-a441-1d0b4263ee99.png)

See that the background is dark and the letter is white? It is the same styling 😄!

Tip for this button:
![image](https://user-images.githubusercontent.com/42799579/205349963-537e9522-26a2-4133-8f3b-ffc3a436c837.png)

You can add a:
```css
border: 0;
```
To make it fit into the original design 😉.